var/spack/repos/builtin/packages/py-breakseq2/package.py

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyBreakseq2(PythonPackage):
+    """nucleotide-resolution analysis of structural variants"""
+
+    homepage = "http://bioinform.github.io/breakseq2/"
+    url      = "https://github.com/bioinform/breakseq2/archive/2.2.tar.gz"
+
+    version('2.2', '6fd5a103c2781717b0b1d0efcbdc17e7')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-biopython@1.65', type='build')
```

Comment:
`py-biopython` is actually needed at build _and_ run-time. See this import:
```
$ grep -R import | grep Bio
breakseq2/biopy/io/Fasta.py:from Bio import SeqIO
```
Also, according to http://bioinform.github.io/breakseq2/, BWA and samtools are needed at run-time. You should make these explicit dependencies of `type='run'`.